common/transformations/camera.py

```diff
@@ -6,8 +6,10 @@
 ## -- hardcoded hardware params --
 eon_f_focal_length = 910.0
 eon_d_focal_length = 650.0
+# tici_f: narrow road cam. tici_e: wide road cam
 tici_f_focal_length = 2648.0
-tici_e_focal_length = tici_d_focal_length = 567.0 # probably wrong? magnification is not consistent across frame
+tici_d_and_e_focal_length = 567.0 # probably wrong? magnification is not consistent across frame
```

Comment:
Do we need this changes? What's wrong with using `tici_e_focal_length` in carla?

Reply:
Not necessary, but thought this would be better for the reader to know that both are the same